src/core/DataSourceBase.ts

```diff
@@ -46,8 +47,8 @@ export abstract class DataSourceBase<TQuery extends DataQuery> extends DataSourc
     }
   }
 
-  get<T>(url: string) {
-    return this.fetch<T>({ method: 'GET', url });
+  get<T>(url: string, params: Record<string, any> | undefined = undefined) {
```

Comment:
This is equivalent but shorter:
```suggestion
  get<T>(url: string, params?: Record<string, any>) {
```

Reply:
I could've swore I had this initially and my editor was complaining about it. Seems to be working now.